src/string/stpncpy.c

```diff
@@ -12,23 +12,27 @@ char *__stpncpy(char *restrict d : itype(restrict _Array_ptr<char>) count(n),
                 size_t n)
   : itype(_Array_ptr<char>) count(n)
 {
+// n and d are used in the declared bounds of d. But both are modified in the loop and hence the declared bounds of d no
```

Comment:
Please indent the comment with the code in this function. Also make sure each line of the comment is no longer than 80 chars.